examples/README.md

```diff
@@ -75,9 +66,11 @@ Sometimes you want to edit existing code examples that is live in the `/examples
 
 1. Edit code examples within the `/examples/<version_number>` directory.
 2. Commit and push the code examples to the repo.
-3. In Github Actions run a manual workflow with the `<version_number>` input of the Handsontable and Handsontable's wrapper version.
 4. The code examples will be deployed to the Github Pages and will be available under the same URL as they already were.
```

Comment:
Please clarify in that the last point that this happens automatically after the merge to `develop`

Reply:
Added in https://github.com/handsontable/handsontable/pull/9165/commits/5a486fa53cea65673c24677f8e069adb5002063a#diff-49aaa2819e35a856818ecec8c9fa7e1c79ad028d3f44bd749736353cfb51bac9R71